src/shared/logs-show.c

```diff
@@ -403,24 +404,27 @@ static int output_timestamp_realtime(
                         break;
 
                 case OUTPUT_SHORT_ISO:
-                        if (strftime(buf, sizeof(buf), "%Y-%m-%dT%H:%M:%S%z",
+                        if (strftime(buf, sizeof(buf), "%Y-%m-%dT%H:%M:%S",
                                      localtime_or_gmtime_r(&t, &tm, flags & OUTPUT_UTC)) <= 0)
                                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
                                                        "Failed to format ISO time");
-                        break;
-
-                case OUTPUT_SHORT_ISO_PRECISE: {
-                        char usec[7];
+                        timezone_off = 19;
+                        goto colonised_timezone;
 
+                case OUTPUT_SHORT_ISO_PRECISE:
```

Comment:
if you intend to define local variables here please open a block right here.

local variables inside of switch() statements are really weird, since their lifetime ignores the case labels, but their initialization cares. thus you might have local variables that exist in further cases down, but are not initialized there.

(this gets particularly weird and confusing with `_cleanup_`)

anyway, our general rule is to avoid this mess: if you want local variables, then open the a {} block right after the `:` of the `case` line